scripts/visualize.py

```diff
@@ -43,59 +42,35 @@ def main(_):
     if random_seed is not None:
         torch.manual_seed(random_seed)
 
-    dataset = meshnets.utils.datasets.FromDiskGeometricDataset(FLAGS.data_dir)
-    size_dataset = len(dataset)
-    num_training = int(FLAGS.train_split * size_dataset)
-    _, validation_dataset = torch.utils.data.random_split(
-        dataset, [num_training, size_dataset - num_training])
-
-    wrapper = meshnets.utils.model_loading.load_model_from_mlflow(
-        FLAGS.tracking_uri, FLAGS.run_id, FLAGS.checkpoint)
-
-    sample_idx = np.random.choice(validation_dataset.indices)
-
-    mesh_path = dataset.get_mesh_path(sample_idx)
-    graph_path = dataset.get_graph_path(sample_idx)
+    dataset = datasets.load_dataset(
+        "inductiva/wind_tunnel",
+        version=FLAGS.dataset_version,
+        split=f"train[-{1 - FLAGS.train_split:.0%}:]",
```

Comment:
Are we fetching from the training dataset for the comparisons provided in the PR?

Reply:
At the moment yes. I will generate similar smaller datasets to be used specifically for this validation/visualizations